assets/src/scss/new-identity/style.scss

```diff
@@ -176,3 +181,7 @@ $palette: (
     }
   }
 }
+
+.logged-in-as a {
+  color: var(--gp-green-800) !important;
```

Comment:
Maybe here instead of `!important` we should write:
```
.logged-in-as a {
  color: var(--gp-green-800);

  &:hover,
  &:visited {
    color: var(--gp-green-800);
  }
}
```
I know it's longer, but it would be easier to override for NROs